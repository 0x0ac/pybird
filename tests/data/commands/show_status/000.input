1000-BIRD 1.3.3
1011-Router ID is 195.69.146.34
 Current server time is 10-01-2012 10:24:37
 Last reboot on 03-01-2012 12:46:40
 Last reconfiguration on 03-01-2012 13:56:40
0013 Daemon is up and running 
