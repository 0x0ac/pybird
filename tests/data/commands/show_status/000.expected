{"router_id": "195.69.146.34", "last_reconfiguration": "2012-01-03T13:56:40", "last_reboot": "2012-01-03T12:46:40"}
